### PR TITLE
PRIME-2824 Site ID Duplicates icon did not show up correctly

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

File: prime-dotnet-webapi/Services/CommunitySiteService.cs [hunk 1]
```diff
@@ -108,6 +116,13 @@ private static void GroupSitesToOrgVisually(PaginatedList<CommunitySiteAdminList
             }
         }
 
+        private async Task<int> GetDuplicatePecCount(int? careSettingCode, string pec, int orgSiteId)
```
Comment (neophyte57) [inline prime-dotnet-webapi/Services/CommunitySiteService.cs:119]: Why is this named `orgSiteId`?

Comment (bergomi02) [reply prime-dotnet-webapi/Services/CommunitySiteService.cs:119]: oh, meant to be original site id (not PEC).

Comment (bergomi02) [reply prime-dotnet-webapi/Services/CommunitySiteService.cs]: Updated the name to be clear

**Review by neophyte57 — APPROVED** [review]
Nice job!

**Pull request merged**